### DISCUSSION TITLE
Cogito <> Akash - Partnership

### DISCUSSION THREAD

**Discussion by dockerised** [akash-network/community#530] [General]
Hello,

I am the founder of Cogito Group, a Kubernetes consultancy based in the UK, specializing in developing secure and platform-agnostic solutions for clients in Web3, finance, and AI sectors. Over the past six months, we have been creating solutions on Akash for our customers, which has prompted me to inquire about the possibility of becoming a formal partner. I’d like to provide a short background on why we believe this partnership could be beneficial:

- Managing providers and ensuring their constant availability presents a significant challenge.
- Customers interested in lending their GPU to the network often face substantial hurdles, requiring in-depth Kubernetes knowledge and specific inbound firewall configurations.
- The emergence and growth of DePin businesses signify an increasing demand for rapid integration of multi-cloud strategies.

To address these challenges, we have developed a turn-key solution that allows customers to connect their GPUs to the Akash network in less than 2 minutes. Our scalable, on-demand Kubernetes expertise enables our clients to quickly leverage multiple cloud environments. Through developing this solution, we have gained extensive experience with your provider architecture and configuration settings, and we are eager to explore potential synergies between our companies.

I look forward to hearing from you!

Kind regards,
George

**Comment by anilmurty** [comment]
hey @dockerised - thanks for starting this discussion. Appreciate the work you've been doing. Some specific questions about. what you said and a separate question/ comment below

specific questions:

> specializing in developing secure and platform-agnostic solutions for clients in Web3, finance, and AI sectors. Over the past six months, we have been creating solutions on Akash for our customers

Can you provide some more details/ clarity around:
1. What solutions you've created in the past 6 months for Web3, finance and AI sectors that use Akash compute?
2. Are you able to provide any references to the customers that you are referring to?

>>we have developed a turn-key solution that allows customers to connect their GPUs to the Akash network in less than 2 minutes

Is there a demo of this solution somewhere? 

Separate comment/ note:
We (OCL+Praetor+community) have identified many specific problems and areas of work for improving onboarding and management of providers on Akash that are called out in (https://github.com/akash-network/community/blob/main/product-strategy/praetor-akash-provider-console-strategy.md) - and we intend to work on those in the coming months. It would be great if any additional work done is cohesive with that and ideallt builds on it (given that PraetorApp is likely to be open sourced soon)

Thanks,
Anil

**Reply by dockerised** [reply]
Hi @anilmurty , thank you for your prompt reply. 

> 1. What solutions you've created in the past 6 months for Web3, finance and AI sectors that use Akash compute?
The easy onboarding solution mentioned above to support the democratisation of GPU resource but also a method to load balance backend providers. 

> 2. Are you able to provide any references to the customers that you are referring to?
No, sadly not.

Regarding Praetor documentation you shared - it states an initiative to ease onboarding so perhaps there's an opportunity to share insights into our solution to help accelerate that. 

Thanks,
George

**Comment by 31trainman** [comment]
I'd be happy to help you on akash discord with any questions you might have etc.

**Reply by dockerised** [reply]
Thank you sounds great, how can I find you there

**Reply by 31trainman** [reply]
Np my discord is 31trainman

Here's a link to Akash discord 

https://discord.gg/akash
